layouts.py

```diff
@@ -299,154 +303,100 @@ def calc_volatility_new(pairs, db, coindata_day):
     return pd.DataFrame(df_all).dropna(how="all")
 
 
-df_new = calc_volatility_new(pairs_new, "n", "n")
+df_new = calc_volatility_new(pairs_new, "n", "n") #Used for Mixed Asset Class Vol Chart
 df_new = df_new[
     df_new.index > datetime.datetime(2020, 1, 15)
 ]  # make it so we only have 2020 data
 
-df = calc_volatility(pairs, "Nothing", "Nothing")
-
+df = calc_volatility(pairs, "Nothing", "Nothing", "1D") #Used for Crypto Vol Chart 
 
 today = datetime.datetime.now(tz=pytz.utc).date()
 xd = today - datetime.timedelta(days=30) 
 #TODO Cyrus possibly change this to april 30th
 
 
+c = xd.strftime("%Y-%m-%d")
 
-def graph_volatility(df, coins, xd):
-    source = "Binance"
-    yaxis_dict = dict(
-        title="Rolling 30-Day Volatility",
-        hoverformat=".2f",
-        ticks="outside",
-        tickcolor="#53585f",
-        ticklen=8,
-        tickwidth=3,
-        tick0=0,
-        tickprefix="                 ",
-    )
-    vols = [14, 30, 90]
-    colors = 4 * [
-        "black",
-        "#033F63",
-        "#28666E",
-        "#7C9885",
-        "#B5B682",
-        "#FEDC97",
-        "#F6AE2D",
-        "#F26419",
-        "#5F0F40",
-        "#9A031E",
-        "#CB793A",
-    ]
+#----------------------------------------------------------Bitcoin Volatility-----------------------------------------------------
 
-    index = 0
-    data = []
-    for i in df.columns:
-        data.append(
-            go.Scatter(
-                x=list(df.index),
-                y=list(df[i]),
-                name=i.split("_", 1)[0],
-                visible=False,
-                line_color=colors[index],
-            )
-        )
-        index += 1
+df_btc = df[['BTC_vol_30', 'BTC_vol_60', 'BTC_vol_7']] #Annualized bitcoin vol chart
 
-    num_coins = len(coins)
+df_btc.columns = ['Ann. 30D Volatility', 'Ann. 60D Volatility', 'Ann. 7D Volatility']
```

Comment:
It would be normal to do this in chronological order ie 7, 30, 60 but I'm sure it is fine. @Cyrusb01

Reply:
I had just done that order because that was the order of the one you sent me